packages/client/hmi-client/src/workflow/tera-workflow.vue

```diff
@@ -856,6 +869,16 @@ const unloadCheck = () => {
 	}
 };
 
+const handleDrilldown = () => {
+	const operatorId = route.query?.operator?.toString();
```

Comment:
It would have been nicer to have put this into the watch, and only pass a `operatorId: string|null` to the function. Keeps the function more versatile and the `route.query` manipulation where it belongs.